### ISSUE TITLE
Update president QC

### ISSUE THREAD

**Issue by hoelzer** [replikation/poreCov#110]
We have now auto builds for `president`:

https://hub.docker.com/repository/docker/rkibioinf/president/tags?page=1&ordering=last_updated

and could update to the newest version. Although I dont think that it will change things the way it is used in `poreCov` - mainly we added bug fixes for many file inputs, ...

**Comment by replikation** [comment]
* does it come with the neccessary tools for nextflow? if so we could of course change to these containers

**Comment by hoelzer** [comment]
yes, it should have all the necessary dependencies: https://gitlab.com/RKIBioinformaticsPipelines/president/-/blob/master/Dockerfiles/deployment/Dockerfile#L3

**Comment by replikation** [comment]
* @hoelzer feel free to add and test that everything remains "fine"

**Comment by hoelzer** [comment]
I will add the current container that we autobuild to the config and we will check that together with the current PR #157
